.github/workflows/ci.yml

```diff
@@ -715,22 +715,17 @@ jobs:
     steps:
       - uses: earthly/actions-setup@v1.0.1
       - uses: actions/checkout@v2
-      -
-        name: Set up QEMU
+        with:
+          # By default, only the latest commit is checked out.
+          # Earthly uses the branch name for tagging,
+          # so we need to explicitly check out the branch.
+          ref: ${{ github.head_ref || github.ref_name }}
```

Comment:
The only case where this is slightly not accurate is if someone wants to re-run the CI on an older commit of that branch. The code we had before would still respect the commit hash, but it would reinsert the branch as pointing to that (possibly older) hash.

Reply:
this makes sense; here's a docs update to mention this: #1853